### ISSUE TITLE
Unknown errors and error masking

### ISSUE THREAD

**Issue by divyenduz** [prisma/prisma2#721]
This is partially implemented, this issue tracks the differences between the implementation and the spec: https://github.com/prisma/specs/tree/master/errors

There might be a separate telemetry spec: https://github.com/prisma/specs/issues/76
Which would make this issue blocked on spec.

**Comment by divyenduz** [comment]
Internal note: closing this one, it will be tracked by the telemetry spec. It is mostly there apart from some wording differences, low priority
